gtest/src/program.rs

```diff
@@ -236,38 +267,44 @@ impl<'a> Program<'a> {
         }
     }
 
+    /// Get the current program from system.
```

Comment:
what is current program etc

Reply:
And its actual doesnt query it. It goes into root crate of the test that uses it and finds its binary, uploading for testing in given system